man/tuned-profiles-spectrumscale-ece.7

```diff
@@ -0,0 +1,59 @@
+.\"/* 
+.\" * All rights reserved
+.\" * Copyright (C) 2015-2020 Red Hat, Inc.
```

Comment:
I thought you are from IBM ;-). Also, I'm not a lawyer but I think the year should be just 2020.

Reply:
I am not sure it is possible to copyright config file on a opensoruced GPL tool but does not matter I add those changes https://github.com/redhat-performance/tuned/commit/7ce1772a7887250e7e038723c04af1c3b0f08a6d

Reply:
I don't see why not. The file doesn't contain anything from Tuned. And so given that you are the author of the file, you (or more likely your employer, per your work contract) are the copyright holder. Which means that you can distribute it in any way you like, even under a non-free license. But by creating this pull request and licensing the file under GPL, you're partially giving up your rights as the copyright holder and you're saying that it can be distributed under the terms of the GPL. And that means that we can incorporate your file in the Tuned project.

Or that's my understanding of it :).